backend/gn_plugin_depobio/blueprint.py

```diff
@@ -0,0 +1,97 @@
+from time import sleep
+
+from flask import Blueprint, abort
```

Comment:
```suggestion
from flask import Blueprint
```

La fonction `abort` n'est finalement pas utilisée